web/sablony/blackarrow/index.xtpl

```diff
@@ -24,7 +24,7 @@
     <div class="paticka">
         <div class="paticka_info">
             <a href="mailto:info@gamecon.cz">info@gamecon.cz</a>
-            <div class="paticka_copy">© 2021 Gamecon, Created by Gamecon</div>
```

Comment:
Narval bych tam dynamický rok, něco jako 
`$t->assign(['letosniRok' => date('Y')]);`

Reply:
opraveno, přidáno, stále musím hloubat jak vlastně to funguje.. musím bojovat :)